services/has-many-getter.js

```diff
@@ -69,6 +71,7 @@ function HasManyGetter(model, association, opts, params) {
     })
     .map(function (record) {
       return new P(function (resolve, reject) {
+        if (!record[params.associationName]) { return resolve(); }
```

Comment:
a warning here would be great to inform the devs that the route is not implemented yet and needs to be.